rigour/text/distance.py

```diff
@@ -55,19 +53,23 @@ def levenshtein_similarity(
         left: A string.
         right: A string.
         max_edits: The maximum number of edits allowed.
-        max_percent: The maximum percentage of edits allowed.
+        max_percent: The maximum fraction of the shortest string that is allowed to be edited.
```

Comment:
Maybe it's just me, but I didn't understand this parameter from the comment, so I tried making it a bit clearer. This should be "if max_percent is 0.5 and the shortest word is 4 characters then the max distance is 2 edits".